### ISSUE TITLE
Prepare Short Accessibility POPM Introduction for P.COM

### ISSUE THREAD

**Issue by marcelbertram** [porsche-design-system/porsche-design-system#866]
**Scope**  
Prepare and conduct a short (around 15 mins) presentation for the product manager / product owner of P.COM to introduce to the topic of accessibility (WHAT), WHY it is important and how to continuously handle it.

**Out of Scope**
- Deep dive education session for dev/design/content

**Notes**  
- Goal is to present it in the PM Sync of PCOM and to convince/educate-
- Content should be re-used for the UX Academy: Don't explain how but introduce the what and why

Subtasks
---
- [x] Create presentation 
- [x] Hold presentation at PO/Sync
